docs/Project-Configuration/Evergreen-Data-for-Analytics.md

```diff
@@ -58,6 +58,151 @@ Daily aggregated statistics for test executions run in Evergreen. Test stats are
+| scheduled\_time                | TIMESTAMP | Time the task is first scheduled.
+| start\_time                    | TIMESTAMP | Time the task started running.
+| finish\_time                   | TIMESTAMP | Time the task completed running.
+| activated_time                 | TIMESTAMP | Time the task was marked as available to be scheduled, automatically or by a developer.
+| dependencies\_met\_time        | TIMESTAMP | Time all dependencies are met, for tasks that have dependencies.
+| last\_heartbeat                | TIMESTAMP | Time of the last heartbeat sent back by the agent.
+| time\_taken                    | BIGINT    | Duration, in nanoseconds, the task took to execute, if it has finished, or how long the task has been running, if it has started.
+| wait\_since\_dependencies\_met | BIGINT    | Duration, in nanoseconds, since the task's dependencies have been met.
+| duration\_prediction           | BIGINT    | How long Evergreen estimates this task will take to run.
+| distro                         | VARCHAR   | ID of the distro this task is for.
+| distro\_aliases                | ARRAY     |  Optional secondary distros that can run the task when they're idle.
+| build\_variant                 | VARCHAR   | Name of the task's build variant.
+| build\_variant\_display\_name  | VARCHAR   | Display name of the task's build variant.
+| agent\_version                 | VARCHAR   | Version of the agent that ran this task.
+| host\_id                       | VARCHAR   | ID of the host that ran this task.
+| host\_create\_details          | VARCHAR   | Information about why host.create failed for this task.
+| pod\_id                        | VARCHAR   | ID of the pod that ran this task.
+| container                      | VARCHAR   | Name of the container configuration for running a container task.
+| container\_options             | ROW       | container_options row that configures the container to run the task.
+| container\_allocated           | BOOLEAN   | Whether a container has been allocated for the task.
+| container\_allocation_attempts | BIGINT    | Number of attempts made to allocate a container.
+| results\_service               | VARCHAR   | Name of the service that stores this task's test results.
+| results\_failed                | BOOLEAN   | Whether any of the task's tests have failed.
+| must\_have_results             | BOOLEAN   | Whether the task will be failed on account of no test results.
+| is\_github\_check              | BOOLEAN   | Whether the task should be considered for mainline github checks.
+| can\_reset                     | BOOLEAN   | Whether the task has successfully archived and is in a valid state to be reset.
+| reset\_when\_finished          | BOOLEAN   | Indicates that a task should be reset once it is finished running.
+| reset\_failed\_when\_finished  | BOOLEAN   | That a display task only restart failed tasks when it's restarted.
```

Comment:
I might word this a bit differently, something like: `Indicates that only the failed execution tasks should be restarted upon restarting a display task. Always "false" for non-display tasks.`

Reply:
this field was removed